spacy/training/loggers.py

```diff
@@ -26,14 +26,36 @@ def setup_table(
     return final_cols, final_widths, ["r" for _ in final_widths]
 
 
+# We cannot rename this method as it's directly imported
+# and used by external packages such as spacy-loggers.
```

Comment:
We probably want to change that in `spacy-loggers` and get the function through the registry instead. (clearly that's not a blocker for this PR though)